django_shop/settings.py

```diff
@@ -10,20 +10,28 @@
 https://docs.djangoproject.com/en/1.8/ref/settings/
 """
 
-# Build paths inside the project like this: os.path.join(BASE_DIR, ...)
 import os
+import ConfigParser
 
-BASE_DIR = os.path.dirname(os.path.dirname(os.path.abspath(__file__)))
+config = ConfigParser.ConfigParser()
+config.read(os.path.join(app.root_path, '../settings.ini')
 
+DATABASE_USER = config.get('database', 'DATABASE_USER')
+DATABASE_PASSWORD = config.get('database', 'DATABASE_PASSWORD')
+DATABASE_ENGINE = config.get('database', 'DATABASE_ENGINE')
+DATABASE_NAME = config.get('database', 'DATABASE_NAME')
 
-# Quick-start development settings - unsuitable for production
-# See https://docs.djangoproject.com/en/1.8/howto/deployment/checklist/
+SECRET_KEY = config.get('secrets','SECRET_KEY')
 
-# SECURITY WARNING: keep the secret key used in production secret!
-SECRET_KEY = '10bot^@*^yq7^u3y$jc$=h7@eb-y%7o=x(rd&#(kbgi7$*%o!a'
+DEBUG = config.getboolean('debug','DEBUG')
+TEMPLATE_DEBUG = config.getboolean('debug','TEMPLATE_DEBUG')
```

Comment:
TEMPLATE_DEBUG is in TEMPLATES option now. See https://docs.djangoproject.com/en/1.8/topics/templates/#django.template.backends.django.DjangoTemplates